manifests/init.pp

```diff
@@ -132,6 +132,12 @@
 #   available, likely results in performance degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should
 #   be done incrementally with benchmarking at each step to determine an optimal value for your deployment.
 #
+# @param worker_ttl
+#   The number of seconds before a pulpcore worker should be considered lost. If undefined, pulpcore will use its default setting.
```

Comment:
I would be very explicit in that it's a setting and which setting it is.
```suggestion
#   Configure Pulpcore's WORKER_TTL setting. This is the number of seconds before a Pulpcore worker should be considered lost. If undefined, Pulpcore will use its default setting.
```

Reply:
Great change, TY